app/src/main/java/amritansh/tripathi/com/searchflix/presentation/navigation/Navigator.kt

```diff
@@ -0,0 +1,28 @@
+package amritansh.tripathi.com.searchflix.presentation.navigation
+
+import amritansh.tripathi.com.searchflix.R
+import amritansh.tripathi.com.searchflix.utils.replaceFragment
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentActivity
+import android.support.v7.app.AppCompatActivity
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class Navigator @Inject constructor() {
```

Comment:
I don't think it's a good choice makes this a Singleton. It depends on the activity context, right? If so, it could be a breach of some unexpected behavior in the future.

Reply:
I understand. I'll make the change.